helper/validation/validation.go

```diff
@@ -106,6 +106,28 @@ func StringLenBetween(min, max int) schema.SchemaValidateFunc {
 	}
 }
 
+// StringMatch returns a SchemaValidateFunc which tests if the provided value
+// matches a given regexp, which must compile or else the function will panic.
```

Comment:
Looks like the implementation has changed, as it takes a compiled Regexp now.

Reply:
Right my bad! Will update this shortly